packages/cli/create-medusa-app/src/utils/nextjs-utils.ts

```diff
@@ -56,15 +56,24 @@ export async function installNextjsStarter({
   }
 
   try {
+    // TODO change back to use create-next-app once Next.js v2 changes land on the main branch
     await execute(
       [
-        `npx create-next-app -e ${NEXTJS_REPO} ${nextjsDirectory}`,
+        `git clone ${NEXTJS_REPO} -b v2 ${nextjsDirectory}`,
         {
           signal: abortController?.signal,
-          env: {
-            ...process.env,
-            npm_config_yes: "yes",
-          },
+          env: process.env,
+        },
+      ],
+      { verbose }
+    )
+    await execute(
+      [
+        `npm install`,
```

Comment:
We currently use yarn in the starter, and ideally we use it here as well (although it might not be installed on some machines, which is the main issue. I guess)

Reply:
Got it, I can do what we do when installing Medusa's dependencies which is try to install with `yarn` first and, if not available, then install with `npm`

Reply:
It might not matter much, it's just the lock file being interpreted correctly that MIGHT cause issues, but in general it shouldn't.

Reply:
Changed it now to try to install with `yarn` first.